code/index.html

```diff
@@ -2,17 +2,83 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <title>Project Name</title>
+    <title>Kreativhang</title>
     <link rel="stylesheet" href="./style.css" />
     <!-- dont forget the meta tag for responsiveness -->
   </head>
   <body>
     <header>
       <!-- If you want a header with a title, logo and a navbar, here is where you should add that. -->
+      <div class="hamburger">
+        <div></div>
+        <div></div>
+        <div></div>
+      </div>
+
+      <nav>
+        <a href="./postkarten.html">Postkarten</a>
+        <a href="./faltkarten.html">Faltkarten</a>
+        <a href="./stickers.html">Stickers</a>
+        <a href="./geschenkpapier.html">Geschenkpapier</a>
```

Comment:
If you just want these to look like links, but not actually link to something, you can write like this:
`<a href="#">Postkarten</a>`